doc/docs/modules/ROOT/pages/quickstart.adoc

```diff
@@ -149,6 +149,22 @@ in case you set this option will have the priority compared to the one defined i
 |``
 |Yes^*^
 
+|`relationship.node.map`
+|If true return `source` and `target` nodes as Map<String, String>, otherwise we flatten the properties by returning
+every single node property as column prefixed by `source` or `target`
+|`true`
+|No
+
+|`relationship.source.labels`
+|List of source node Labels separated by `:`
+|``
+|Yes^*^
+
+|`relationship.target.labels`
+|List of target node Labels separated by `:`
+|``
+|Yes^*^
+
 |`schema.flatten.limit`
 |Number of records to be used to create the Schema (only if APOC are not installed)
```

Comment:
I don't really understand what this means.  Does it constraint the maximum number of columns that can come back on a flattened DF?   If I set a limit and something has 40 properties, I guess I lose the ones over the limit but I can't tell which ones I lose, which is problematic.

Reply:
Let me explain, in case APOC are not installed we use these two queries:

For nodes:
```scala
val query = s"""MATCH (${Neo4jUtil.NODE_ALIAS}:${labels.mkString(":")})
                |RETURN ${Neo4jUtil.NODE_ALIAS}
                |ORDER BY rand()
                |LIMIT ${options.query.schemaFlattenLimit}
                |""".stripMargin
```
For relationships:
```scala
val query = s"""MATCH (${Neo4jUtil.RELATIONSHIP_SOURCE_ALIAS}:${options.relationshipMetadata.source.labels.mkString(":")})
                |MATCH (${Neo4jUtil.RELATIONSHIP_TARGET_ALIAS}:${options.relationshipMetadata.target.labels.mkString(":")})
                |MATCH (${Neo4jUtil.RELATIONSHIP_SOURCE_ALIAS})-[${Neo4jUtil.RELATIONSHIP_ALIAS}:${options.relationshipMetadata.relationshipType}]->(${Neo4jUtil.RELATIONSHIP_TARGET_ALIAS})
                |RETURN ${Neo4jUtil.RELATIONSHIP_ALIAS}
                |ORDER BY rand()
                |LIMIT ${options.query.schemaFlattenLimit}
                |""".stripMargin
```

and we use the result of these in order to retrieve the schema, so as we cannot dump the entire node/relationship set we provide the `schema.flatten.limit` in order to sample a fixed number of results. So we're are not defining the number of columns but the number of Neo4j results to parse in compute the schema.

Reply:
Consider naming this `schema.sample` and then including this comment.   Literally you're defining a sample size, right?